SU-HSDO/suhumsci
# shs-10: gradient hero fix animated text bug

## Summary
This fixes a bug with the Hero Gradient Slider where every slide after the first wouldn't animate (fade in) the text.

## Urgency
_'low'_

## Screenshot
![Screen Shot 2022-05-04 at 2 36 02 PM](https://user-images.githubusercontent.com/8405274/166803236-dff018d7-395a-4fd6-87f2-df1a3ea97e78.png)

![Screen Shot 2022-05-04 at 2 36 16 PM](https://user-images.githubusercontent.com/8405274/166803250-db537be7-2be7-4a9e-abfc-23cd55fc3f43.png)

## Steps to Test
1. Go to Drupal and create a Flexible Page. For the 'Full-Width Region' add a 'Hero (Gradient) - Slider' (screenshot above). Add a couple slides.
2. View the page. When clicking on the next or previous slide, verify that the text fades in.
3. Verify this is only happening with `hb-has-animation-enhancements` present (it's in the HTML class list).
